features/scripts/assemble-fixtures.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 
-mkdir -p features/fixtures/libs
-./gradlew bugsnag:assemble bugsnag-spring:assemble -Pversion=9.9.9-test
-cp bugsnag/build/libs/bugsnag-9.9.9-test.jar features/fixtures/libs/bugsnag-9.9.9-test.jar
-cp bugsnag-spring/build/libs/bugsnag-spring-9.9.9-test.jar features/fixtures/libs/bugsnag-spring-9.9.9-test.jar
+if [ ! -d "features/fixtures/libs" ]; then
```

Comment:
required to prevent re-building test fixtures if they were already built on a different system. i.e. copying from java17 builder image to java 8 maze-runner image